util/build_docs.py

```diff
@@ -47,18 +47,24 @@
     SRCTREE_TOP,
 
     # Pre-generate register and hwcfg fragments from these files.
+    #
+    # Commented-out lines are the unparameterised versions of parameterisable IPs,
+    # they are followed by their parameterisations for Earl Grey.
     "hardware_definitions": [
         "hw/ip/aes/data/aes.hjson",
-        "hw/ip/alert_handler/data/alert_handler.hjson",
+        # "hw/ip/alert_handler/data/alert_handler.hjson",
+        "hw/top_earlgrey/ip/alert_handler/data/autogen/alert_handler.hjson",
         "hw/ip/entropy_src/data/entropy_src.hjson",
         "hw/ip/flash_ctrl/data/flash_ctrl.hjson",
         "hw/ip/gpio/data/gpio.hjson",
         "hw/ip/hmac/data/hmac.hjson",
         "hw/ip/i2c/data/i2c.hjson",
         "hw/ip/nmi_gen/data/nmi_gen.hjson",
         "hw/ip/padctrl/data/padctrl.hjson",
-        "hw/ip/pinmux/data/pinmux.hjson",
-        "hw/ip/rv_plic/data/rv_plic.hjson",
+        # "hw/ip/pinmux/data/pinmux.hjson",
```

Comment:
if we're OK with these changes, we should remove the comments.

Reply:
agreed